pages/treeherder.py

```diff
@@ -51,6 +59,42 @@ def open_perfherder_page(self):
         from perfherder import PerfherderPage
         return PerfherderPage(self.base_url, self.selenium).wait_for_page_to_load()
 
+    def select_next_job(self):
+        el = self.selenium.find_element(*self._resultset_locator)
+        Wait(self.selenium, self.timeout).until(EC.visibility_of(el))
+        el.send_keys(Keys.ARROW_RIGHT)
+        Wait(self.selenium, self.timeout).until(lambda s: self.job_result_status)
```

Comment:
That is true as the job details area remains open. I had no luck finding a way to ensure that the new job title is not the same as the previous job title that isn't super complicated. Is it worth checking the current title, moving to the next job and ensuring that the title has changed?